dotnet/Workloads/SignList.targets

```diff
@@ -16,20 +16,23 @@
     <Unzip
         SourceFiles="@(_NestedBrokerZip)"
         DestinationFolder="@(_NestedBrokerZip -> '$(_NestedZipExtractionDir)%(Filename)')"
+        Condition="'@(_NestedBrokerZip->Count())' != '0'"
     />
-    <Delete Files="@(_NestedBrokerZip)" />
+    <Delete Files="@(_NestedBrokerZip)" Condition="'@(_NestedBrokerZip->Count())' != '0'" />
```

Comment:
Was this one actually a problem? If the file doesn't exist, mine does:

![image](https://user-images.githubusercontent.com/840039/192419615-5c43af55-0c1e-4620-adba-1a7d35f405b2.png)

No need to fix this, if we need this in quick.

Reply:
fails on our side complaining in the UnitStep that is missing a value for the DestinationFile.

Reply:
So `<Delete Files="@(SomeEmptyItemGroup)" />` fails, ok I see.

Reply:
Hmm, I don't know, this works fine for me:

![image](https://user-images.githubusercontent.com/840039/192591463-2c0d11f2-c72d-44ca-93d4-5f4099fab135.png)

Reply:
Ah I think you are right - the conditions on the delete tasks were probably not needed. Should be harmless either way as you mentioned though.